controllers/home_controller.rb

```diff
@@ -11,25 +11,6 @@ class HomeController < ApplicationController
     get do
       expires 3600, :public
       last_modified @@root_last_modified ||= Time.now.httpdate
-      routes = routes_list
```

Comment:
here you don't use anymore theses functions: `routes_list` , `routes_by_class`, so need to be removed also

Reply:
they are removed no ?

Reply:
you mean remove them from home_helper.rb ?